homeworks/hw25/pages/cart_page.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=import-error
```

Comment:
If you want to disable it permanently - add it to this file https://github.com/ValeryiaDomokurova/HomeWork4/blob/main/.github/workflows/pylint.yaml